src/Compilers/Shared/BuildServerConnection.cs

```diff
@@ -13,6 +13,7 @@
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
+using System.Web;
```

Comment:
>System.Web [](http://example.com/codeflow?start=6&length=10)

Is this used?

Reply:
This feedback about unused usings was used.